generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -60,13 +82,18 @@ public void setLastModifiedDate(ZonedDateTime lastModifiedDate) {
         this.lastModifiedDate = lastModifiedDate;
     }<% } %>
 
+    public String getPassword() {
+        return password;
+    }
+
     @Override
     public String toString() {
         return "ManagedUserDTO{" +
             "id=" + id +<% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
             ", createdDate=" + createdDate +
             ", lastModifiedBy='" + lastModifiedBy + '\'' +
-            ", lastModifiedDate=" + lastModifiedDate +<% } %>
+            ", lastModifiedDate=" + lastModifiedDate + '\'' +<% } %>
+            ", password='" + password +
```

Comment:
I don't think it's a good idea to append the encrypted password in the toString method, it could easily be printed in application logs by mistake.